context/ifc-context.tsx

```
@@ -457,42 +457,64 @@ export function IFCContextProvider({ children }: { children: ReactNode }) {
+                const pset = itemProps.propertySets[name];
+                if (pset && pset[propName] !== undefined) {
+                  elementValue = pset[propName];
                   break;
                 }
               }
             }
 
+            // If not found via propertySets above, use existing fallback logic
+            if (elementValue === undefined) {
+              let psetObject: any = undefined;
+              if (itemProps && itemProps[psetPattern]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)